rpc/rpc.go

```diff
@@ -57,6 +57,10 @@ const (
 	TaggedCmdAcknowledge         = "ack"
 	TaggedCmdPing                = "ping"
 	TaggedCmdPong                = "pong"
+
+	// misc
+	MessageModeNormal = 0
```

Comment:
sure, i'm just more interesting in having a different type here to prevent the accidental usage of unrecognized modes.